docs/concepts/amino.md

```diff
@@ -0,0 +1,3 @@
+# Amnio Encoding
```

Comment:
This shouldn't be a whole section ideally.

Reply:
I don't know, amino is not really well-known. And it's more a section about encoding in general than amino, I will update.